### PR TITLE
Troubleshooting: Add details about ping and arping

### PR DESCRIPTION
**Pull request by Mtze**
Added details to the troubleshooting page I learned during a debugging session with @fedepaol.

### CODE REVIEW

File: website/content/configuration/troubleshooting.md [hunk 1]
```diff
@@ -39,7 +43,9 @@ Received 4 response(s)
 
 `192.168.1.35` is the IP of one of the worker nodes and part of the same subnet.
 
-### tcpdump
+> Make sure to use `arping` on a node which is not homing the `metallb-speaker` that ARPs for the address. ARP requests from the same host will be ingored.
```
Comment (fedepaol) [inline website/content/configuration/troubleshooting.md:46]: typo: ingored

File: website/content/configuration/troubleshooting.md [hunk 2]
```diff
@@ -20,7 +20,11 @@ assignment of the IP and can't connect.
 
 ## Tools
 
-### arping
+### `ping`
```
Comment (fedepaol) [inline website/content/configuration/troubleshooting.md:23]: Can you add this as a note (look for the other examples):
```
{{% notice note %}}

```

Also, I'd put this in the bottom of the file

Comment (fedepaol) [reply website/content/configuration/troubleshooting.md:23]: Also, replace OS interface with host interface

**Comment by fedepaol** [comment]
@Mtze mind squashing the commits instead of appending new ones (unless they serve a different purpouse)?

**Comment by Mtze** [comment]
Hmm I updated the branch from main between my commits  - How would I rebase my feedback commit into the first commit? Not sure if that is even possible. Wouldn't it be easier to squash the commits on merge?

**Comment by fedepaol** [comment]
> Hmm I updated the branch from main between my commits - How would I rebase my feedback commit into the first commit? Not sure if that is even possible. Wouldn't it be easier to squash the commits on merge?

You can rebase on main, then rebase -i and squash the commits.

**Comment by Mtze** [comment]
Ok - should be ok now :) And I learned something new again :)

**Comment by fedepaol** [comment]
LGTM, thanks and sorry for the delay!

**Pull request merged**